project4/index.html

```diff
@@ -0,0 +1,248 @@
+                <li><a href="#speakers" class="click left">Speakers</a></li>
+                <li><a href="#schedules" class="click left">Schedule</a></li>
+                <li><a href="#tickets" class="click left">Tickets</a></li>
+            </ul>
+        </nav>
+
+        <a href="#">
+            <svg class="big" id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 27.94 39.5">
+                <defs>
+                    <style>
+                        .cls-1 {
+                            fill: none;
+                            stroke: #fff;
+                            stroke-width: 0.5px;
+                        }
+                    </style>
+                </defs>
+                <title>gdl</title>
+                <g id="Group_17" data-name="Group 17">
+                    <path id="Path_11" data-name="Path 11" class="cls-1"
+                        d="M277.23,384.21a9,9,0,0,1,4.7-4.45,8.41,8.41,0,0,1,3.29-.63,8.25,8.25,0,0,1,4,1,8.9,8.9,0,0,1,3.12,2.79l1.62-1.23a10.26,10.26,0,0,0-1.72-1.94,11.42,11.42,0,0,0-2.1-1.49,10.74,10.74,0,0,0-2.39-1,10.23,10.23,0,0,0-6.64.49,11.36,11.36,0,0,0-3.37,2.27,11.88,11.88,0,0,0-2.37,3.34,11,11,0,0,0-1.07,4h2.09A8.36,8.36,0,0,1,277.23,384.21Z"
+                        transform="translate(-274.03 -376.75)" />
+                    <path id="Path_12" data-name="Path 12" class="cls-1"
+                        d="M300.81,392.55a11.68,11.68,0,0,0-2.44-3.57,11.4,11.4,0,0,0-3.59-2.43,10.7,10.7,0,0,0-4.36-.89h-6.17v12.08a8.74,8.74,0,0,1-1.26-.23v-4h-2.07v3.11a9,9,0,0,1-3.59-3.7,8.22,8.22,0,0,1-.93-3.13h-2.1a10.94,10.94,0,0,0,1.14,3.87,11.51,11.51,0,0,0,5.48,5.28v4.79H283v-4.08a10.34,10.34,0,0,0,1.26.2v3.83h2.09v-3.85a10.94,10.94,0,0,0,1.39-.25,10.31,10.31,0,0,0,2.37-.92,10.74,10.74,0,0,0,2.1-1.48,12.79,12.79,0,0,0,1.77-2v-5.41h-7.63v-2.08h4.54a7.94,7.94,0,0,1,3.46.76,9.3,9.3,0,0,1,2.81,2,9.71,9.71,0,0,1,1.89,2.92,8.75,8.75,0,0,1,.7,3.42,8.28,8.28,0,0,1-.43,2.75,10.36,10.36,0,0,1-1.28,2.47,8.33,8.33,0,0,1-1.79,1.91,9.08,9.08,0,0,1-2.11,1.17,10.75,10.75,0,0,1-2.35.63,22.89,22.89,0,0,1-2.48.25c-.84,0-1.69.05-2.53,0l-2.51,0v2.07H291a9.65,9.65,0,0,0,4.19-.94,11.49,11.49,0,0,0,3.43-2.52,12.2,12.2,0,0,0,2.3-3.61,10.92,10.92,0,0,0,.84-4.19A10.7,10.7,0,0,0,300.81,392.55Zm-8.93-.67v2.76a9.44,9.44,0,0,1-3.11,2.36,8.61,8.61,0,0,1-2.43.72v-5.84Z"
+                        transform="translate(-274.03 -376.75)" />
+                    <path id="Path_13" data-name="Path 13" class="cls-1" d="M283,406.11h-2.07V416H294.4v-2.07H283Z"
+                        transform="translate(-274.03 -376.75)" />
+                </g>
+            </svg>
+        </a>
+
+        <a href="#" class="login">Login</a>
+
+        <img class="neon" src="images/asset.png" alt="gdl neon">
+
+
+        <h1 class="gdl">X—Future on</h1>
+        <h1 class="gdlg">GDL 2020</h1>
+
+        <p class="gdlp">Share your artworks with artists</p>
+
+        <button class="learn">LEARN MORE</button>
```

Comment:
So, some of the jumping around I saw in your layout comes from the changing values for this absolutely positioned button. You're using `top` to position this, but that becomes a moving target as images and other things resize. I recommend 2 things:
1. Adding a wrapper element around lines 61-66 (I originally went with a `header`, but you are already using that and have styles on it, so `div` would work, too). Add a classname to that `div` and make the `div` `position: relative;` in your CSS.
1. Instead of making the "Learn More" `button` positioned via `top`, you can remove _all_ the `top` values for this class in your CSS and now instead position it via `bottom: 0;` relative to the wrapper `div`.

When I make that change, the resizing is much smoother:

![2019-12-12 13 47 19](https://user-images.githubusercontent.com/2653980/70740155-86551480-1ce6-11ea-9d32-9674af77d4b3.gif)

If there are other absolutely positioned items, giving them more local/nearby elements to position against, might make things feel less jumpy. Hope this helps!